plugins/modules/create_blueprint.py

```diff
@@ -104,12 +113,13 @@ def main():
             dest=dict(type="str", required=True),
             name=dict(type="str", required=True),
             description=dict(type="str", required=False, default=""),
-            version=dict(type="str", required=False, default="0.0.1"),
+            version_type=dict(type="str", required=False, default="patch"),
```

Comment:
This needs to include the `choices=["patch", ...]` list and it should match the same entry in the DOCUMENTATION string.

Example: https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/yum.py#L67 and https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/yumdnf.py#L49